src/SFA.DAS.LevyTransferMatching.Web/Models/Opportunities/GetFundingEstimateViewModel.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace SFA.DAS.LevyTransferMatching.Web.Models.Opportunities
+{
+    public class GetFundingEstimateViewModel
+    {
+        public double Amount { get; set; }
```

Comment:
There were a few places I could have put this comment, but I chose here. The Amount is declared as a double, but I believe once calculated it is rounded and treated as an integer (both when displayed on-screen, but also when stored in the database). This is why I raised the question of what rounding we should use.